Police-Data-Accessibility-Project/data-source-map
# feat: add search overlay box

Resolves #9 

## To test
Pull branch, run `npm run dev`, you should experience something like this:


https://github.com/Police-Data-Accessibility-Project/data-source-map/assets/68428039/430f120f-1c6c-4f1b-a950-9c5a115ad90d
